File: app/src/main/java/de/christinecoenen/code/zapp/utils/system/MenuHelper.java
package de.christinecoenen.code.zapp.utils.system;

import android.view.Menu;
import android.view.MenuItem;

public class MenuHelper {

	public static void uncheckItems(Menu menu) {
		if (menu == null) {
			return;
		}

		for (int i = 0; i < menu.size(); i++) {
			MenuItem item = menu.getItem(i);
			item.setChecked(false);
			uncheckItems(item.getSubMenu());
		}
	}

}

File: app/src/main/java/de/christinecoenen/code/zapp/utils/system/MenuHelper.kt
package de.christinecoenen.code.zapp.utils.system

import android.view.Menu
import androidx.core.view.forEach

object MenuHelper {

	fun uncheckItems(menu: Menu) {
		menu.forEach { item ->
			item.isChecked = false
			uncheckItems(item.subMenu)
		}
	}
}
